nemo/collections/asr/models/rnnt_models.py

```diff
@@ -120,6 +123,32 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             self._optim_variational_noise_std = 0
             self._optim_variational_noise_start = 0
 
+        # Setup normalized gradients for model
+        if 'normalize_joint' in self.cfg:
+            self._optim_normalize_joint = self.cfg.normalize_joint
+            self._optim_normalize_txu = None
+        else:
+            self._optim_normalize_joint = False
+            self._optim_normalize_txu = None
+
+        # Setup normalized encoder norm for model
+        if 'normalize_encoder_norm' in self.cfg:
```

Comment:
These four parameters, if they are similar and related, how about adding a new section like optim_normalization and add them under that? Just a suggestion to organize them. It can also make the names smaller too.

Reply:
Yes we can rename this function to `_setup_optim_normalization`

Reply:
I meant adding a section to the config file like this:
model:
    optim_normalization:
         normalize_encoder_norm: true
         normalize_joint:false
    ...

the current form is also OK, just a suggestion to make it more organized as they are at least 4 and maybe more in future may be added.

Reply:
Yes that can be another option. Other than variational noise, I expect the other 3 to be rarely used. They only really apply to very specific combination of model and optimizer so we might refactor them into their own section if they every become more useful in general case